data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -11,6 +11,15 @@ bootkube_podman_run() {
     podman run --quiet --net=host "${@}"
 }
 
+copy_static_resources_for() {
```

Comment:
define this function in the context of the mco only.

Reply:
done